specs/player_spec.rb

```diff
@@ -0,0 +1,182 @@
+require 'minitest/autorun'
+require 'minitest/reporters'
+require 'minitest/skip_dsl'
+
+require_relative '../lib/player'
+require_relative '../lib/scoring'
+require_relative '../lib/tilebag'
+
+Minitest::Reporters.use! Minitest::Reporters::SpecReporter.new
+
+describe 'Player' do
+  describe 'initialize' do
+    it "new player is an instance of class Player" do
+      Scrabble::Player.new("Ana").must_be_instance_of Scrabble::Player
+    end
+
+    it "player name is a kind of string" do
+      Scrabble::Player.new("Ana").name.must_be_kind_of String
+    end
+
+    it "player begins with no tiles, no words played, and a score of zero" do
+      player_1 = Scrabble::Player.new("Ana")
+
+      player_1.words_played.must_equal []
+      player_1.tiles.must_equal []
+      player_1.score.must_equal 0
+    end
+  end
+
+  describe 'plays' do
+    it "returns an array" do
+      player_1 = Scrabble::Player.new("Wini")
+
+      player_1.plays.must_be_kind_of Array
+    end
+
+    it "returns an array of words played" do
+      player_1 = Scrabble::Player.new("Wini", true)
+      player_1.play("hello")
+      player_1.play("hi")
+
+      player_1.plays.must_equal ["HELLO", "HI"]
+    end
+  end
+
+  describe 'play' do
+    it "adds the played word to the array of words played" do
+      player_1 = Scrabble::Player.new("Dan", true)
+      player_1.play("xx")
+
+      player_1.words_played.must_include "XX"
+    end
+
+    it "the score does not change if the user enters an invalid word" do
+      player_1 = Scrabble::Player.new("Dan", true)
+      score_before = player_1.score
+      player_1.play('')
+      player_1.play(555)
+      player_1.play('@^&')
+      score_after = player_1.score
+
+      score_after.must_equal score_before
+    end
+
+    it "returns the score of the word if the game is not won" do
+      player_1 = Scrabble::Player.new("Dan", true)
+
+      player_1.play("xx").must_equal 16
+    end
+
+    it 'returns false if the game is won' do
+      player_1 = Scrabble::Player.new("Dan", true)
+
+      player_1.play("zzzzffd").must_equal 100
+      player_1.play("xx").must_equal false
+    end
+  end
+
+  describe "total score" do
```

Comment:
It would also be a good idea to test that total score doesn't go up if the player keeps playing past winning.